packages/test/test-end-to-end-tests/src/test/gc/gcDatastoreAliased.spec.ts

```diff
@@ -51,7 +51,7 @@ describeFullCompat("GC Data Store Aliased", (getTestObjectProvider) => {
         runtimeOptions,
     );
 
-    // Enable config provider setting to write GC data at the root.
+    // Enable config provider settings.
```

Comment:
Please add what config you are enabling or remove this comment. Currently, it doesn't provide any meaningful info.